### PR TITLE
Remove debug tags from netlify build

### PR DESCRIPTION
**Pull request by nickvigilante**

### CODE REVIEW

File: netlify/build [hunk 1]
```diff
@@ -23,10 +23,8 @@ cp _site/docs/404.html _site/404.html
 
 # Set up htmltest
 
-curl https://htmltest.wjdp.uk | bash
-echo "debug 0"
+curl https://htmltest.wjdp.uk | bash > /dev/null 2>&1
```
Comment (mdlinville) [inline netlify/build:26]: This will fail if that site is down but you won't find out because you're redirecting STDERR ... that OK?

**Review by mdlinville — APPROVED** [review]

File: netlify/build (hunk 1 above)
Comment (nickvigilante) [reply netlify/build]: I fixed it in the event that we can't download htmltest.

File: netlify/build [hunk 2]
```diff
@@ -23,10 +23,12 @@ cp _site/docs/404.html _site/404.html
 
 # Set up htmltest
 
-curl https://htmltest.wjdp.uk | bash
-echo "debug 0"
+curl -s https://htmltest.wjdp.uk | bash
+if [[ $? != 0 ]]; then
+	echo "Failed to install htmltest"
+	exit 1
+fi;
```
Comment (mdlinville) [inline netlify/build:30]:
```suggestion
fi
```
It will work but it's non-standard syntax.

**Review by mdlinville — APPROVED** [review]

File: netlify/build (hunk 2 above)
Comment (nickvigilante) [reply netlify/build:30]: There are other parts of this file that have it, so I'll log a new PR for it on Monday.

**Pull request merged**